infra/vm/scripts/restore-longevity.sh

```
@@ -0,0 +1,133 @@
+    exit 1
+fi
+
+# --- Pick one ---
+printf '\nEnter the backup name to restore from (Ctrl-C to abort): ' >&2
+read -r BACKUP
+[ -n "${BACKUP:-}" ] || { echo "No backup name entered; aborting." >&2; exit 1; }
+
+# Validate the named backup exists and is Completed before we touch anything.
+# Use kubectl for the lookup — the velero CLI's `-o` only does table/json/yaml,
+# not jsonpath, so we query the Backup CRD directly.
+PHASE=$(ssh "$VM" "$REMOTE_KUBECTL -n velero get backups.velero.io '$BACKUP' -o jsonpath='{.status.phase}'" 2>/dev/null || true)
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)